src/cmd/cmd_sym.ml

```diff
@@ -251,6 +251,11 @@ let handle_result ~exploration_strategy ~workers ~no_stop_at_failure ~no_value
       ~count_acc:0 ~results ~with_breadcrumbs
   in
   Log.info (fun m -> m "Completed paths: %d" (Atomic.get path_count));
+
+  let bench_stats = Thread_with_memory.bench_stats thread in
```

Comment:
Can you move this inside the second `Log.bench` so that it is not executed when not in benchmarking mode?